frontend/src/component/layout/MainLayout/MainLayout.tsx

```diff
@@ -74,9 +78,33 @@ const MainLayoutContentContainer = styled('div')(({ theme }) => ({
     zIndex: 200,
 }));
 
+const SegmentsSplash: React.FC = () => {
+    const { value: localStorageState, setValue: setLocalStorageState } =
+        createLocalStorage('OssSegmentsSplashScreen:v1', { shown: false });
+
+    const [showSegmentSplash, setShowSegmentSplash] = React.useState(true);
+
+    const navigate = useNavigate();
+    const closeSegmentsSplash = () => {
+        setShowSegmentSplash(false);
+        setLocalStorageState({ shown: true });
+    };
+
+    return (
+        <SegmentsSplashScreen
+            open={showSegmentSplash && !localStorageState.shown}
+            onClose={closeSegmentsSplash}
+            showSegments={() => {
+                closeSegmentsSplash();
+                navigate(`/segments`);
+            }}
+        />
+    );
+};
```

Comment:
I feel like this belongs in a separate file and not part of MainLayout. WDYT?

Reply:
Yeah, that's fair. I put it here just for now, but I could pop it in the SegmentsSplashScreen directory. Should probably be part of the main exported component, I reckon. What do you think?

Reply:
I think that could work, definitely better than being a part of `MainLayout` 👍